README.md

```diff
@@ -169,14 +172,142 @@ The command above will create three pods associated to the existing `imagestream
 objects that point to `openstack`, `openshift` and `kubevirt` must-gather container
 images.
 
-```
+```bash
 [must-gather] OUT pod for plug-in image quay.io/openstack-k8s-operators/openstack-must-gather:latest created
 [must-gather] OUT pod for plug-in image quay.io/kubevirt/must-gather@sha256:501e30ac7d5b9840a918bb9e5aa830686288ccfeee37d70aaf99cd2e302a2bb0 created
 [must-gather] OUT pod for plug-in image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e9601b492cbb375f0a05310efa6025691f8bba6a97667976cd4baf4adf0f244c created
 ...
 ...
 ```
 
+### `sos report` from nodes
+
+To gather config files and logs from hosts we need to rely on the `sos` tool (former `sosreport`).
+
+We can obtain logs from both OCP cluster nodes and compute nodes.
```

Comment:
For OCP cluster nodes the SOS report should be gathered using the mechanism that is being added in [PR#19](https://github.com/openstack-k8s-operators/openstack-must-gather/pull/19), though I think it has value to document the manual mechanism here.